site/content/docs/5.3/examples/download-app/index.html

```diff
@@ -52,7 +52,7 @@ <h1 class="two-lined">
 <main>
   <div class="text-bg-dark mb-0 mb-md-2 border-dark border-bottom border-1 position-relative">
     <div class="col-12 col-lg-7 out-of-grid h-100 end-0 overflow-hidden">
-      <img loading="lazy" class="h-100 object-fit-cover" alt="" src="img/Banner_image.png">
+      <img loading="lazy" class="h-100 object-fit-cover" alt="Orande Design System, winner of the 2023 Red Dot Award for Interface Design" src="img/Banner_image_reddot.png">
```

Comment:
```suggestion
      <img loading="lazy" class="h-100 object-fit-cover" alt="Orange Design System App, winner of the 2023 Red Dot Design Award for User Interface and User Experience within the category of Brand and Communication Design" src="img/Banner_image_reddot.png">
```

I've added more details in the alternative text than in the actual image, hope it's not an issue.